integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("(Integration) [non-eksctl cluster & nodegroup support]", func(
 	})
 
 	AfterSuite(func() {
-		if !params.SkipDelete {
+		if !params.SkipCreate || !params.SkipDelete {
```

Comment:
| SkipCreate       | SkipDelete           | deleteStack  |
| ------------- |:-------------:| -----:|
| false      | false | true |
| false     | true      |   false |
| true |  false   |    false/true? | 
| true |  true   |    false |

Reply:
We don't have a stack to delete if SkipCreate is true, so the 3rd row outcome is `false`

Reply:
ok so then it should be `!params.SkipCreate && !params.SkipDelete` :dart:

Reply:
oops :+1: